routes/index.js

```diff
@@ -264,43 +268,60 @@
                 errors: results.array()
             });
         } else {
-            const verificationID = nanoid();
+            console.log(req.body)
+            
+            
+            db.registerUser(req.body.userName, req.body.institute_id, req.body.email, req.body.password, req.body.mobile, req.body.address, req.body.city, req.body.country, req.body.state, req.body.postcode, req.body.photo)
+            .then((data)=>{
+                console.log(data)
+                res.send(' Signup Sucessful.');
+            }).catch(err=>{
```

Comment:
Codacy has a fix for the issue: [Expected parentheses around arrow function argument.](https://app.codacy.com/gh/StudentConnects/Mentorspi-Interaction-System/pullRequest?prid=7436658)
```suggestion
            }).catch((err)=>{
```